bundle.yaml

```diff
@@ -17,7 +17,7 @@ applications:
   landscape-server:
     charm: ch:landscape-server
     channel: stable
-    revision: 111
+    revision: 124
```

Comment:
This is the specific rev that fixes this or just the latest rev?

Reply:
It was the latest one a at the time